tests/Feature/Api/Assets/AssetUpdateTest.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+namespace Tests\Feature\Api\Assets;
+
+use App\Models\Asset;
+use App\Models\AssetModel;
+use App\Models\Company;
+use App\Models\Location;
+use App\Models\Statuslabel;
+use App\Models\Supplier;
+use App\Models\User;
+use Tests\Support\InteractsWithSettings;
+use Tests\TestCase;
+
+class AssetUpdateTest extends TestCase
+{
+    use InteractsWithSettings;
```

Comment:
#14438 just got merged so this can go away if there is more work done before merging. I don't think there is an issue with having the trait doubled-up so it can get cleaned up in the future too.